### ISSUE TITLE
Add content about GSoC (google summer of code)

### ISSUE THREAD

**Issue by bluddy** [OCamlverse/ocamlverse.github.io#30]
One of the very big things OCaml is constantly missing out on is google summer of code participants. In order to get the ball rolling, we need a place to organize a list of projects or ideas that could be contributed to. We also need to have a template ready for our GSoC application.

There was some work done in the past on the ocaml.org wiki. It should be imported and embellished.

The part about ideas for projects could also be generalized -- it doesn't just need to be for GSoC.

**Comment by pmetzger** [comment]
Who would they work for, though? Is there a central OCaml project that can act as to GSoC sponsor?

**Comment by bluddy** [comment]
Yeah you really don't need much. Just having them work for the OCaml organization is good enough. I've done this in the past with a different open source organization. The only things you need are:

1. Good project suggestions
2. A decent looking application (it really doesn't need to be fancy)
3. People willing to mentor for the summer. I've mentored before, and while it takes up time, it's not that demanding.
4. A procedure in-place to check that the intern is making good progress.

The intern gets paid for the summer by Google, and hopefully contributes something worthwhile to the community.

**Comment by pmetzger** [comment]
> Just having them work for the OCaml organization is good enough. 

I'm still quite fuzzy on what the OCaml organization actually is. 😃

**Comment by loxs** [comment]
The OCaml github organization: https://github.com/ocaml

**Comment by pmetzger** [comment]
A github organization isn't the same as a formal group of people.

**Comment by bluddy** [comment]
Oh also, to check that a participant is serious, organizations usually require something first. The participant needs to propose something good, but they also need to contribute to the organization in some minor way e.g. fixing a bug or something.

**Comment by denis631** [comment]
Out of curiosity, is there any progress on this one?
